src/utilities.lisp

```diff
@@ -167,36 +167,34 @@ The result will be a list of cons cells representing half-open intervals (on the
   #-unix
   1)
 
-(let ((prepared? nil)
-      (workers-allotted nil))
-  (defun prepare-for-parallelization (&optional num-workers)
-    "Prepare for parallelization with the correct number of workers scaling with the number of logical cores of your machine.
+(defun prepare-for-parallelization (&optional num-workers)
+  "Create a worker pool if none exists or if NUM-WORKERS has changed.
 
-If NUM-WORKERS is provided, it can force the number of workers. If it's greater than 1, then it should be less than the number of logical cores of your machine.
+If NUM-WORKERS is not provided, the number of workers will be set to the number of logical cores of your machine. ~
+This function does nothing if NUM-WORKERS workers have already been created. ~
```

Comment:
Whoops, forgot to save this comment before clicking "finish review":

Nitpick: I have not seen the trailing-tilda convention used elsewhere in rigetti documentation strings. Not saying there is anything wrong with it, just that I haven't noticed it in the `quilc` codebase, and I assume conventions are similar over here in QVM-land.

To be clear: I am just a random contributor, not a code owner. This only needs to change if the maintainers say so :).

Reply:
You are probably correct. I'm not familiar with this codebase, either. I'll try to make this consistent with the other doc strings.

Reply:
Doc strings are not format strings, and shouldn't have the `~`

Reply:
Convention is also to not wrap long lines in doc strings.

Reply:
(I think though that putting `\` at the end has the effect you want in the string-literal context.)

Reply:
Got it.